be/src/exec/vectorized/orc_scanner_adapter.cpp

```diff
@@ -1196,6 +1196,15 @@ Status OrcScannerAdapter::_init_cast_exprs() {
             _cast_exprs[column_pos] = slot;
             continue;
         }
+        // we don't support implicit cast column in query external hive table case.
+        // if we query external table, we heavily rely on type match to do optimization.
+        // For example, if we assume column A is a integer column, but it's stored as string in orc file
+        // then min/max of A is almost unusable. Think that there are values [10, 11, 10000, 100001]
+        // min/max will be "10" and "11", and we expect min/max is 10/100001
+        if (!_broker_load_mode) {
```

Comment:
The mode means multi option，but we only have is load or is query. So I think the better name is `_is_broker_load`

Reply:
Yes, `is_broker_load` is a better name.